templates/commit_details.erb

```diff
@@ -0,0 +1,16 @@
+[GitHub] Commit <%= @commit['id'][0,Lita::Handlers::GithubCommits::SHA_ABBREV_LENGTH]%> committed by <%=@commit['committer'] ? @commit['committer']['name'] : "<unknown>"%> on branch <%=@commit['branch']%> at <%=Time.parse(@commit['timestamp']).getlocal.to_s%> with message
+  '<%=@commit['message']%>'
+<%if !@commit['added'].empty?%>
+Files Added
+  <%=@commit['added'].join("\n  ")%>
+<%end%>
+<%if !@commit['removed'].empty?%>
+Files Removed
+  <%=@commit['removed'].join("\n  ")%>
+<%end%>
```

Comment:
Doesn't this add in a bunch of extra newlines when you put the `if` and `end` on new lines? I know it probably doesn't matter for Slack/Hipchat, but this probably would look bad on IRC.